models/staging/commissioning/sus/stg_sus_ecds_clinical_comorbidities.sql

```
@@ -6,4 +6,4 @@ select primarykey_id
     ,comorbidities_id
     ,rownumber_id
     ,code
-from {{ ref('raw_sus_ae_clinical_comorbidities') }}
\ No newline at end of file
+from {{ ref('raw_sus_ecds_clinical_comorbidities') }}
```

SQLFluff (4.2.2)

[error] Files must end with a single trailing newline.

(LT12)